consoleApp/src/main/java/com/infoshare/database/FileDb.java

```diff
@@ -0,0 +1,112 @@
+package com.infoshare.database;
+
+import com.infoshare.domain.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileDb implements DB {
+    private static final String VOLUNTEER_DB_FILE_NAME = "Volunteer.csv";
```

Comment:
Przynajmniej u mnie - jeżeli pliku nie mamy (np. pierwsze uruchomienie programu) to się wywala wyjątek FileNotFoundException. Wydaje mi się, że brakuje warunku na sprawdzenie czy plik istnieje. Jeżeli nie, tworzymy nowy.

Reply:
metodą prób i błędów (zakomentowywania i odkomentowywania :D) wyszło mi że problem jest w metodzie `getVolunteer` , która jest użyta jako początkowy warunek przy zapisie. Ja spróbowałam w metodzie `getVolunteer` dodać na początku coś takiego (można chyba też try/catch):
```
 File file = new File(VOLUNTEER_DB_FILE_NAME);
        if(!file.canRead()){
            return null;
        }
        Scanner scanner = new Scanner(file);
```
Po tej poprawce 'saveVolunteer' zadziałało :+1:

Reply:
dodalem sprawdzene czy plik istnieje